packages/angular-workspace/package.json

```diff
@@ -41,7 +41,7 @@
     "@ni/spright-components": "*",
     "rxjs": "^7.4.0",
     "tslib": "^2.2.0",
```

Comment:
@fredvisser Do you know why this PR was created? Are you investigating using Snyk instead of or in addition to Renovate?

I don't think we want this particular update because zone.js is a dependency that we typically upgrade in lockstep with the Angular major version.